src/pytest_plugins/execute/sender.py

```diff
@@ -104,7 +89,7 @@ def sender_key_initial_balance(
             assert seed_sender_balance_per_worker > 100, "Seed sender balance too low"
             # Subtract the cost of the transaction that is going to be sent to the seed sender
             sender_key_initial_balance = seed_sender_balance_per_worker - (
-                sender_fund_refund_gas_limit * sender_funding_transactions_gas_price
+                sender_fund_refund_gas_limit * default_gas_price
```

Comment:
sender_funding_transactions_gas_price I think that we should keep previous logic for supporting passing the price funding, the only change only need to pump refunding gas price from funding gas price ?

Reply:
Returned the parameter, but it now defaults to the same value as `--default-gas-price`, and we still bump the gas price for the refunds.